pkg/filters/scope.go

```diff
@@ -160,9 +163,26 @@ func (f *ScopeFilter) Parse(field string, operatorAndValues string) error {
 		filter.Enable()
 		return filter.add(false, Equal)
 	case "container":
-		filter := f.containerFilter
-		filter.Enable()
-		return filter.add(true, Equal)
+		// Handle container state filtering: container, container=started
+		// All container filters require containerFilter to be enabled
+		f.containerFilter.Enable()
+		if err := f.containerFilter.add(true, Equal); err != nil {
+			return errfmt.WrapError(err)
+		}
+
+		if operatorAndValues == "" {
+			// Just "container" - any container (ID != "")
+			return nil
+		}
+
+		// Parse container=started
+		if operatorAndValues == "=started" {
+			// Only started containers (ContainerStarted == true)
+			f.containerStartedFilter.Enable()
+			return f.containerStartedFilter.add(true, Equal)
+		}
+
+		return InvalidExpression(operatorAndValues)
```

Comment:
@yanivagman sorry, I might be missing context but from the PR description I couldn't understand why we also adding a userland filter for it, if we have a ebpf filter. Shouldn't this be like `container=new` filtered directly in ebpf, without the need for another filter here?